app/styles/search-page.less

```diff
@@ -55,4 +55,35 @@
       min-height: 50px;
     }
   }
+
+  .search-grid {
+    .school-grid-row {
+      .row();
+      margin-top: 30px;
+      padding-bottom: 40px;
+      .school-grid-wrapper {
+        .make-column(4, 4);
+
+        .school-grid {
+          padding: 10px 2px 0px 2px;
+          border-bottom: 1px solid @border-main-color;
```

Comment:
A slight transition for the background-color would be nice here:
`.transition(background-color 0.3s);`
Fancier animations and such can be added later.